docker-compose.yaml

```diff
@@ -2,11 +2,11 @@
 # Copyright 2021 New Relic Corporation. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 #
-version: '3.7'
+version: '3.8'
 services:
 # The Database
   mysqldb:
-    image: mysql:5.7
+    image: mysql:5.6
```

Comment:
Since there aren't official mysql images for arm64, consider forcing this image's platform to be linux/amd64 by like this:
```
   mysqldb:
-    image: mysql:5.7
+    platform: linux/amd64
+    image: mysql:5.6
```
With this change it will be possible to compose images and run containers on arm64 machine with mysql container running in emulation mode. But maybe this change does not belong to this PR.